DEHPEcosimPro/ViewModel/DifferenceViewModel.cs

```diff
@@ -0,0 +1,140 @@
+//    License as published by the Free Software Foundation; either
+//    version 3 of the License, or (at your option) any later version.
+// 
+//    The DEHPEcosimPro is distributed in the hope that it will be useful,
+//    but WITHOUT ANY WARRANTY; without even the implied warranty of
+//    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+//    Lesser General Public License for more details.
+// 
+//    You should have received a copy of the GNU Lesser General Public License
+//    along with this program; if not, write to the Free Software Foundation,
+//    Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+// </copyright>
+// --------------------------------------------------------------------------------------------------------------------
+
+
+namespace DEHPEcosimPro.ViewModel
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Reactive.Linq;
+
+    using Autofac;
+    using CDP4Common.EngineeringModelData;
+
+    using CDP4Dal;
+    using DEHPCommon.HubController.Interfaces;
+    using DEHPEcosimPro.DstController;
+    using DEHPEcosimPro.Events;
+    using DEHPEcosimPro.ViewModel.Interfaces;
+    using DEHPEcosimPro.ViewModel.Rows;
+
+    using NLog;
+
+    using ReactiveUI;
+
+
+    /// <summary>
+    /// The <see cref="DifferenceViewModel"/> is the view model for displaying diference betwen the values of the selection
+    /// </summary>
+    public class DifferenceViewModel : ReactiveObject, IDifferenceViewModel
+    {
+        /// <summary>
+        /// The <see cref="IHubController"/>
+        /// </summary>
+        private readonly IHubController hubController;
+
+        /// <summary>
+        /// The <see cref="IDstController"/>
+        /// </summary>
+        private readonly IDstController dstController;
+
+        /// <summary>
+        /// List of parameter to show on the window
+        /// </summary>
+        public ReactiveList<ParameterDifferenceRowViewModel> Parameters { get; set; } = new ReactiveList<ParameterDifferenceRowViewModel>();
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="T:DEHPEcosimPro.ViewModel.DifferenceViewModel" /> class.
+        /// </summary>
+        /// <param name="hubController"><see cref="IHubController"/></param>
+        /// <param name="dstController"><see cref="IDstController"/></param>
+        public DifferenceViewModel(IHubController hubController, IDstController dstController)
+        {
+            this.hubController = hubController;
+            this.dstController = dstController;
+
+            CDPMessageBus.Current.Listen<DifferenceEvent<ParameterOrOverrideBase>>()
+                .Subscribe(this.HandleDifferentEvent);
+
+
+            CDPMessageBus.Current.Listen<DifferenceEvent<ElementDefinition>>()
+                .Subscribe(this.HandleListOfDifferentEvent);
+
+        }
+
+        /// <summary>
+        /// Handle the parameter
```

Comment:
Please be more descriptive in comments. If I read this description and look at the method I don't correlate.

Reply:
also in the <param> do not just put the class of the thing that is being argumented.